### ISSUE TITLE
Adding ELPD

### ISSUE THREAD

**Issue by ParadaCarleton** [JuliaML/LossFunctions.jl#144]
Maximizing the expected log posterior density (ELPD) is a common objective function in Bayesian inference and Bayesian neural networks, and I'd be interested in having it as a loss function for a package I'm building. I'd be happy to implement it if someone around here is willing to help me figure out the API (I haven't used JuliaML before; I'm building a package that does efficient LOO-CV for Bayesian models).

(Note: ELPD is the generalization of log-loss/cross-entropy loss to regression, rather than classification, problems.)

**Comment by juliohm** [comment]
@ParadaCarleton please draft a PR and we can discuss details.

**Comment by juliohm** [comment]
Closing as inactive. Happy to discuss in a new issue based on the current state of the package.
